content/influxdb/v2.0/write-data/developer-tools/csv.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: Write CSV data to InfluxDB
 description: >
-  Use the [`influx write` command](/influxdb/v2.0/reference/cli/influx/write/) to write CSV data
-  to InfluxDB. Include annotations with the CSV data to determine how the data translates
-  into [line protocol](/influxdb/v2.0/reference/syntax/line-protocol/).
+  Write CSV data with the following methods:
+  - [flux](#flux)
+  - [influx write command](#influx-write-command)
```

Comment:
To keep the description on one line in the final HTML, how about:

```suggestion
Write CSV data with the [`influx write` command](#influx-write-command) and Flux.
Include annotations with the CSV data to determine how the data translates into
[line protocol](/influxdb/v2.0/reference/syntax/line-protocol/).
```

Reply:
Thanks for the help.